<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorConfigWidgetForm</class>
 <widget class="QWidget" name="MotorConfigWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="motorEnabledCheckBox">
     <property name="text">
      <string>Motor Enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="inputParametersGroupBox">
       <property name="title">
        <string>Input Parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="nStepsPerTurnLabel">
          <property name="text">
           <string>Full steps per turn</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="nStepsPerTurn">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="maxRPMLabel">
          <property name="text">
           <string>Maximum velocity </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="maxRPM">
          <property name="suffix">
           <string> RPM</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="timeToVMaxLabel">
          <property name="text">
           <string>Time to max. velocity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="timeToVMax">
          <property name="suffix">
           <string> s</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>9.999900000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="microstepsLabel">
          <property name="text">
           <string>Microsteps</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="microsteps">
          <property name="currentIndex">
           <number>4</number>
          </property>
          <item>
           <property name="text">
            <string>1 (fullstep)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 (halfstep)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">64</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="systemClockLabel">
          <property name="text">
           <string>System clock</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="systemClock">
          <property name="suffix">
           <string> MHz</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QCheckBox" name="positiveSwitchCheckBox">
          <property name="text">
           <string>Positive end switch enabled</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCheckBox" name="negativeSwitchCheckBox">
          <property name="text">
           <string>Negative end switch enabled</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="iMax">
          <property name="suffix">
           <string> A</string>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>1.800000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.240000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="iMaxLabel">
          <property name="text">
           <string>Max. coil current</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>nStepsPerTurnLabel</zorder>
       <zorder>nStepsPerTurn</zorder>
       <zorder>maxRPMLabel</zorder>
       <zorder>maxRPM</zorder>
       <zorder>timeToVMaxLabel</zorder>
       <zorder>microstepsLabel</zorder>
       <zorder>systemClockLabel</zorder>
       <zorder>systemClock</zorder>
       <zorder>positiveSwitchCheckBox</zorder>
       <zorder>negativeSwitchCheckBox</zorder>
       <zorder>microsteps</zorder>
       <zorder>timeToVMax</zorder>
       <zorder>iMax</zorder>
       <zorder>iMaxLabel</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="chipParametersGroupBox">
       <property name="title">
        <string>Chip Parameters </string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QLabel" name="pDivDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="pMulDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">174</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="driverMicrostepValueDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="vMaxDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">1398</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="currentScaleDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="aMaxDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">1466</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="pDivLabel">
          <property name="text">
           <string>pDiv</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="pMulLabel">
          <property name="text">
           <string>pMul</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="controllerMicrostepValueLabel">
          <property name="text">
           <string>controllerMicrostepValue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="pulseDivLabel">
          <property name="text">
           <string>pulseDiv</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="vMaxLabel">
          <property name="text">
           <string>vMax</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="driverMicrostepValueLabel">
          <property name="text">
           <string>driverMicrostepValue</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="aMaxLabel">
          <property name="text">
           <string>aMax</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="rampDivLabel">
          <property name="text">
           <string>rampDiv</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="currentScaleLabel">
          <property name="text">
           <string>currentScale</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="controllerMicrostepValueDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="rampDivDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">11</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="pulseDivDisplay">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="warningsGroupBox">
     <property name="title">
      <string>Conversion Warnings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="warningsBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>motorEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputParametersGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motorEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>warningsGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motorEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>chipParametersGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
